docs/understanding-your-pipeline/canonical-event/index.md

```diff
@@ -42,280 +48,403 @@ The sections below go over the standard fields found in all Snowplow events.
+| `se_category` | text     | Category of event                                                                              | Yes*       | 'ecomm', 'video'              |
+| `se_action`   | text     | Action performed / event name                                                                  | Yes*       | 'add-to-basket', 'play-video' |
+| `se_label`    | text     | The object of the action e.g. the ID of the video played or SKU of the product added-to-basket | No        | 'pbz00123'                    |
+| `se_property` | text     | A property associated with the object of the action                                            | No        | 'HD', 'large'                 |
+| `se_value`    | decimal  | A value associated with the event / action e.g. the value of goods added-to-basket             | No        | 9.99                          |
 
-:::
 
-If you wish to track an event that Snowplow does not recognise as a first class citizen (i.e. one of the events listed above), then you can track them using the generic 'custom structured events'. There are five fields that are available to store data related to custom events:
+\* These fields are only required for `struct` events.
+
+#### Custom unstructured (self-describing) events
+
+Custom [unstructured (self-describing) events](/docs/understanding-tracking-design/out-of-the-box-vs-custom-events-and-entities/index.md#self-describing-events) are a flexible tool that enable Snowplow users to define their own event types and send them into Snowplow.
 
-| **Field** | **Type** | **Description** | **Reqd?** | **Example** |
-| --- | --- | --- | --- | --- |
-| `se_category` | text | Category of event | Yes | 'ecomm', 'video' |
-| `se_action` | text | Action performed / event name | Yes | 'add-to-basket', 'play-video' |
-| `se_label` | text | The object of the action e.g. the ID of the video played or SKU of the product added-to-basket | No | 'pbz00123' |
-| `se_property` | text | A property associated with the object of the action | No | 'HD', 'large' |
-| `se_value` | decimal | A value associated with the event / action e.g. the value of goods added-to-basket | No | 9.99 |
+When a user sends in a custom unstructured event, they do so as a JSON of name-value properties that conforms to a JSON schema defined for the event earlier.
+
+<Tabs groupId="warehouse" queryString>
+<TabItem value="rs/pg" label="Redshift/Postgres" default>
 
-#### Custom unstructured events
+The unstructured event is not part of the standard `atomic.events` table, instead, for users running on Redshift, it is shredded into its own table. The fields in this table will be determined by the JSON schema defined for the event in advance. Users can query just the table for that particular unstructured event, if that's all that's required for their analysis, or join that table back to the `atomic.events` table by
```

Comment:
Maybe instead of "shredded" use "transformed by the RDB Loader" and link to https://docs.snowplow.io/docs/destinations/warehouses-and-lakes/rdb/ or something? Seems like we're moving away from that term too.